third_party/rust/chromium_crates_io/Cargo.lock

```diff
@@ -1145,6 +1309,9 @@ dependencies = [
 [[package]]
 name = "lazy_static"
 version = "1.4.0"
+dependencies = [
```

Comment:
this change is not correct, there should not be any new deps for `lazy_static` as this is an upstream crate and the version has not changed